server/routes/status.js

```diff
@@ -2,38 +2,36 @@ export default (app) => {
   app
     .get('/status', async (req, reply) => {
       const statuses = await app.objection.models.status.query();
-      await reply.render('status/index', { currentUser: req.currentUser, statuses });
+      await reply.render('status/index', { data: { currentUser: req.currentUser, statuses } });
     })
     .get('/status/new', async (req, reply) => {
-      await reply.render('status/new', { data: {}, errors: [] });
+      await reply.render('status/new', { data: { status: { name: '' } }, errors: {} });
     })
     .get('/status/edit/:id', async (req, reply) => {
       const status = await app.objection.models.status.query().findById(req.params.id);
-      await reply.render('status/edit', { status, errors: [] });
+      await reply.render('status/edit', { data: { status }, errors: {} });
     })
     .post('/status', async (req, reply) => {
       try {
-        const newStatus = await app.objection.models.status.fromJson(req.body);
+        const newStatus = app.objection.models.status.fromJson(req.body);
         await app.objection.models.status.query().insert(newStatus);
         await reply.redirect('/status');
       } catch ({ data }) {
-        await reply.code(400).render('status/new', { data: req.body, errors: data });
+        await reply.code(400).render('status/new', { data: { status: req.body }, errors: data });
       }
     })
     .put('/status', async (req, reply) => {
```

Comment:
for patial update rest offers method PATCH
https://medium.com/backticks-tildes/restful-api-design-put-vs-patch-4a061aa3ed0b

Reply:
done